gnovm/tests/files/types/0f21_filetest.gno

```diff
@@ -0,0 +1,13 @@
+package main
```

Comment:
it looks like github doesn't syntax highlight files that start with a number, so please move these to some descriptive prefix_number.

Reply:
done.